hw/xbox/nv2a_vsh.h

```diff
@@ -24,6 +24,25 @@
 
 #include "qapi/qmp/qstring.h"
 
+enum Texgen {
```

Comment:
Probably, yeah. Don't worry too much about it, we'll make a pass refactoring the vsh and psh stuff eventually.

Thanks!